src/components/MenuPanel/MenuPanel.js

```diff
@@ -274,7 +274,7 @@ class MenuPanel extends React.PureComponent {
     return (
       <div key={appId}>
         <MenuPanelAppGroup
-          name={name}
+          name={menuName || name}
```

Comment:
name / alias sounds better, what do you think?

Reply:
Totally agree.